### DISCUSSION TITLE
[Script request] Bitwarden Server

### DISCUSSION THREAD

**Discussion by serpyxa** [community-scripts/ProxmoxVE#211] [Request script] [answered]
### Application Name

Bitwarden

### Website

https://github.com/bitwarden/server

### Description

Self hosted password manager

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by jwgn** [comment, marked as answer]
Why not use Vaultwarden?

https://community-scripts.github.io/ProxmoxVE/scripts?id=vaultwarden
https://github.com/dani-garcia/vaultwarden

**Comment by SoTHISIsFitness** [comment]
Isn't this what Vaultwarden is for? https://community-scripts.github.io/ProxmoxVE/scripts?id=vaultwarden

**Comment by pictosun** [comment]
As we do now have Bitwarden Lite it would be nice to maybe rethink it as it gives us a chance to install Bitwarden on Proxmox.

See here https://bitwarden.com/help/install-and-deploy-lite/

**Reply by MickLesk** [reply]
For what? Its an Lite docker Version. Vaultwarden is enough

**Reply by pictosun** [reply]
Vaultwarden is having issues, isn't fully compatible (at the moment) and needs more development. So it's always good to have an alternative in my opinion. And the main developer from Vaultwarden also went to Bitwarden a while ago... So why not?

**Reply by MickLesk** [reply]
Docker.

**Reply by tremor021** [reply]
> Vaultwarden is having issues, isn't fully compatible (at the moment) and needs more development. So it's always good to have an alternative in my opinion. And the main developer from Vaultwarden also went to Bitwarden a while ago... So why not?

what kind of issues?

**Reply by pictosun** [reply]
See here: https://github.com/dani-garcia/vaultwarden/issues

**Reply by tremor021** [reply]
yes, but its like every software. issues happen, issues gets fixed...
Same as our own github issues

I'm just insterested in what kind of issue it has that stops you from using it. I never encountered such issue
